frontend/src/locales/es/translation.json

```diff
@@ -2,5 +2,6 @@
   "LastUpdatedLabel": "Última actualización",
   "SearchButton": "Buscar",
   "ClearSearchText": "Limpiar parámetros de búsqueda",
-  "AllDashboardsLink": "Todos los dashboards"
+  "AllDashboardsLink": "Todos los dashboards",
+  "ArchivedTabDescription": "to be translated"
```

Comment:
Since we don't have the translation yet, a better approach would be to not include this key at all in the Spanish json file. That way it will default to English instead of showing a `to be translated` string.